.github/workflows/codeQL-analysis.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: Static analysis with CodeQL
```

Comment:
Let's keep name shorts

* rename file to `codeql.yaml`
and: 
```suggestion
name: codeql
```

Reply:
Done